scripts/filter-known-issues.py

```diff
@@ -147,42 +147,42 @@ def config_import(paths):
 exclude_ranges = []
 
 if args.warnings:
-    warnings = open(args.warnings, "w")
+    warnings = open(args.warnings, "wb")
```

Comment:
The same question. The idea is simple - if a file context text information, it should be open as text (like it was), if it contains binary data - as binary. I doubt that args.warnings refers to a binary file.